includes/class_plugins.php

```diff
@@ -796,6 +807,31 @@ function trigger($event_name, array $vars)
 		return $vars;
 	}
 
+	/**
+	* Returns the hook files for the plugins.
+	*/
+	function get_hook_files($hook)
+	{
+		global $config;
+
+		$files = array();
+		$base_dir = '../../' . PLUGINS_PATH;
```

Comment:
Why you didn't use IP_ROOT_PATH instead of '../../'?

Reply:
Because the template class doesn't know how to handle `IP_ROOT_PATH`. I tried it.